server/etc/httpd/conf.d/pulp.conf

```diff
@@ -35,9 +35,11 @@ WSGISocketPrefix run/wsgi
 WSGIScriptAlias /pulp/api /usr/share/pulp/wsgi/webservices.wsgi
 WSGIImportScript /usr/share/pulp/wsgi/webservices.wsgi process-group=pulp application-group=pulp
 
-<Directory /usr/share/pulp/wsgi>
-    Require all granted
-</Directory>
+<IfVersion >= 2.4>
+    <Directory /usr/share/pulp/wsgi>
```

Comment:
Not directly related to this review: any idea why there wasn't a corresponding block in the 2.2 config? Obviously the WSGI app works on 2.2, so maybe this block isn't required at all? Or maybe there's some new behavior in 2.4 that requires it? A comment would be helpful if we can figure out why this is here.